sql/wsrep_sst.cc

```diff
@@ -772,6 +773,20 @@ std::string wsrep_sst_prepare() {
     return WSREP_STATE_TRANSFER_TRIVIAL;
   }
 
+  if (!strcmp(wsrep_sst_method, WSREP_SST_ONLY_IST)) {
+    /* Inform Galera that we are done with SST and it can proceed with IST.
+    In fact the following call will wait until the server is fully initialized
+    and then inform Galera about two things:
+    1. call sst_received() - so from Galera's point of view it looks like we are done with
+                             sst
+    2. return empty string from this function - informs Galera that only IST should
+       be processed.*/
+    WSREP_WARN("State Transfer via SST was prohibited by setting wsrep_sst_method=ist_only. "
+               "The node will try to join the cluster using only IST.");
+    wsrep_sst_complete(current_thd, 0);
+    return "";
```

Comment:
I would have a `define` to represent this empty string, just like `WSREP_STATE_TRANSFER_TRIVIAL` or `WSREP_STATE_TRANSFER_NONE`